app/frontend/courses/curricula/components/CoursesCurriculum__SubmissionsAndFeedback.res

```diff
@@ -51,17 +51,13 @@ let submissionStatusIcon = (~passed) => {
   let text = passed ? tr("completed") : tr("rejected")
   let color = passed ? "green" : "red"
 
-  <div className="max-w-fc">
-    <div
-      className={"flex justify-center border-2 rounded-lg border-" ++ (color ++ "-500 px-4 py-6")}>
-      {passed
-        ? <div className="fa-stack text-green-500 text-lg">
-            <i className="fas fa-certificate fa-stack-2x" />
-            <i className="fas fa-check fa-stack-1x fa-inverse" />
-          </div>
-        : <i className="fas fa-exclamation-triangle text-3xl text-red-500 mx-1" />}
-    </div>
-    <div className={"text-center text-" ++ (color ++ "-500 font-bold mt-2")}> {text |> str} </div>
+  <div className="max-w-fc flex items-center">
+    {passed
+      ? <span className="flex text-green-500 text-lg me-2">
+          <Icon className="if i-badge-check-solid text-2xl" />
+        </span>
+      : React.null}
+    <p className={"text-center text-sm text-" ++ (color ++ "-700 font-medium")}> {text |> str} </p>
```

Comment:
Tailwind won't generate CSS for `color ++ "-700 "`

Reply:
@irajsuhail Updated here https://github.com/pupilfirst/pupilfirst/pull/1597/commits/83109a65a0b5e1569c8c7398951001d12d98ebad